prpr/src/judge.rs

```diff
@@ -472,13 +472,12 @@ impl Judge {
                     {
                         continue;
                     }
-                    let dt = dt
-                        + if matches!(note.kind, NoteKind::Flick | NoteKind::Drag) {
-                            0.05 /* TODO tweak*/
+                    let dt = if matches!(note.kind, NoteKind::Flick | NoteKind::Drag) {
+                            if dt <= LIMIT_PERFECT {dt.max(LIMIT_PERFECT)} else {dt.max(LIMIT_GOOD)}
```

Comment:
Could you please elaborate on this?

Reply:
modified, nvm that

Reply:
It's a failed conjecture of the original mechanisms.